webapp-django/crashstats/topcrashers/jinja2/topcrashers/topcrashers.html

```diff
@@ -164,8 +164,29 @@
                                 <input type="hidden" class='ajax-signature' name="ajax-signature-1" value="{{ crash.signature }}" />
                             </div>
                             <div class="signature-icons">
+                                {% if crash.startup_count > 0 %}
+                                {% if crash.startup_count == crash.count %}
```

Comment:
Perhaps we should be a bit more flexible here and give it a red flag if 90% or 95% or 99% of the crashes have startup_crash=True.
So something like `crash.startup_count >= 0.95 * crash.count`.
Otherwise even if there's just one outlier, we will show the yellow flag.